jujugui/static/gui/src/app/app.js

```diff
@@ -764,7 +768,12 @@ YUI.add('juju-gui', function(Y) {
         });
       });
 
-      controllerAPI.after('connectedChange', e => {
+      controllerAPI.after('connectedChange', evt => {
+        if (!evt.newVal) {
```

Comment:
This is when the network disconnects (connected != authenticated). In here we want to react to changes to the connection only when the change means that it reconnected. We already have a comment there: 
`// The controller is not connected, do nothing waiting for a reconnection.`